### DISCUSSION TITLE
Connector timeout error when I use promise.all

### DISCUSSION THREAD

**Discussion by jsun969** [prisma/prisma#9455] [Q&A]
My schema is
```prisma
datasource db {
  provider = "sqlite"
  url      = "file:./db.db"
}

model Friend {
  id          Int      @id @default(autoincrement())
  createdAt   DateTime @default(now())
  name        String
  address     String
  description String
  avatar      String
  order       Int      @default(0)
}
```
and I want to update with a array include ids and orders like
```ts
const orders=[{id:2, order:5}, {id:3, order:4}]
```
This is how I update data
```ts
await Promise.all(
  orders.map(({ id, order }) => {
    return prisma.friend.update({
      where: { id },
      data: { order },
    });
  })
);
```
I got error
```
Error: 
Invalid `prisma.friend.update()` invocation:
Error occurred during query execution:
ConnectorError(ConnectorError { user_facing_error: None, kind: ConnectionError(Timed out during query execution.) })
```

**Comment by ryands17** [comment]
@jsun969 👋 

It would be great if you could create an issue [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml) with the above details so that we can look into this.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
